components/styles.tsx

```diff
@@ -1,14 +1,22 @@
-import { StyleSheet } from 'react-native';
+import { StyleSheet, Dimensions } from 'react-native';
+import { colors } from '../style/styleVariables';
 
 const styles = StyleSheet.create({
   properties: {
-    color: 'white',
+    color: colors.light,
     fontSize: 16
   },
   propRow: {
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'space-between'
+  },
+  container: {
```

Comment:
could we slowly move out of this file and put `styles` next to components? Feel free to do that in a separate PR though